lib/tasks/test.rake

```diff
@@ -0,0 +1,5 @@
+desc "Run sass-lint"
+task :sass_lint do
+  system("node_modules/.bin/sass-lint -vq")
+  puts "No sass linting errors!  Woo!"
```

Comment:
This message is emitted regardless of the exit status of sass-lint. You need something like

    system("...") && puts("...")

or

    puts "..." if $?.success?